TODO.md

```diff
@@ -0,0 +1,17 @@
+
+
+# Temporary file, to log todos
+
+
+one pr for multiple image changes (?) need to handle this case later on
```

Comment:
Agreed one PR for multiple image change for less PR spam :D

Reply:
Agreed, the options would be either one or both at the same time, then reject the un-necessary PR.

I think we can go to prod with just one PR for each image update.  We can implement this nice to have in subsequent phase.